test/test_warp_scan.cu

```diff
@@ -399,7 +401,10 @@ void Test(
     InitialValueT   initial_value)
 {
     enum {
-        TOTAL_ITEMS = LOGICAL_WARP_THREADS * NUM_WARPS,
+        IS_ARCH_WARP = (LOGICAL_WARP_THREADS == CUB_WARP_THREADS(0)),
```

Comment:
What is CUB_WARP_THREADS supposed to do?

Reply:
It represents hardware warp size throughout the project.